index.html

```diff
@@ -1,442 +1,641 @@
-                        <li class="dropdown submenu">
-                           <a href="services.html" class=""> Services</a>
-                        </li>
-                        <li class="dropdown submenu">
-                           <a href="blog.html" class="">Blog</a>
-                        </li>
-                        <li class="dropdown submenu">
-                           <a href="elements.html" class="">Elements</a>
-                        </li>
-                        <li class="dropdown submenu">
-                           <a href="contact.html" class="">Contact</a>
-                        </li>
-                      
-                     </ul>
-                        </div>
-                        <!-- /.navbar-collapse -->
-                     </div>
-                  </div>
-                  <!-- /.container -->
-               </nav>
-               <!-- End Header_Area -->
-               <!-- #banner start -->
-               <section id="banner" class=" mb-90">
-                  <div class="container ">
-                     <div class="row">
-                        <!-- #banner-text start -->            
-                        <div id="banner-text" class="col-md-12 text-c text-center ">
-                           <h5 class="wow fadeInUp main-h" data-wow-delay="0.2s" >End-to-End Customer Journey Analytics</h5>
-                           <p class="banner-text wow fadeInUp main-h3" data-wow-delay="0.8s">No hours sank into aggregating and cleaning data. No complex SQL queries required. Just the answers <br> teams need to make smarter decisions, fast. Now, that's data-driven.</p>
-                           <div class="top-banner wow fadeInRight">
-                              <a id="#services"  href="contact.html" class="btn btn-default  wow fadeInUp  js-scroll-trigger" data-wow-delay="1s" href="#">GeT STARTED FOR FREE</a>
-                           </div>
-                        </div>
-                        <!-- /#banner-text End -->
-                     </div>
-                  </div>
-               </section>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <!-- Always include this line of code!!! -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Rubik:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="assets/css/general.css" />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <link rel="stylesheet" href="assets/css/queries.css" />
+
+    <script
+      type="module"
+      src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"
+    ></script>
+    <script
+      nomodule=""
+      src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.js"
+    ></script>
+
+    <title>Tisamed</title>
```

Comment:
Please add all meta-tags required for good SEO and social media previews(facebook, twitter, linkedin) using this tool: https://metatags.io/ to generate the metatags and paste them here, in the <head> section.

Use the following info: 
title="Tisamed"
description="Cabinet medical obstetrică-ginecologie Tisamed cu vastă experienţă acumulată în domeniul medical, vă oferă consultant medicala la cea mai înaltă calitate."
image="https://i.ibb.co/TKy86tw/tisamed.jpg"